zerver/context_processors.py

```diff
@@ -105,6 +106,9 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
         settings_path = "/etc/zulip/settings.py"
         settings_comments_path = "/etc/zulip/settings.py"
 
+    support_email = FromAddress.SUPPORT
+    support_email_html_tag = SafeString(f'<a href="mailto:{support_email}">{support_email}</a>')
```

Comment:
I don't think SafeString is correct for this -- if the email address, which in the below case is somewhat user-controlled, contains HTML characters, it could be a problem.  Or is that being escaped elsewhere?

Reply:
My reasoning was that the email would be marked as invalid during email change if it contains HTML tags.

eg

![Screenshot from 2020-09-17 00-42-46](https://user-images.githubusercontent.com/7190633/93381785-cd2bf800-f87e-11ea-8771-0c774cfb466e.png)

Also, the template is only shown to the user itself and nobody else since this is a confirmation page. Would that be an issue?

I

Reply:
> Also, the template is only shown to the user itself and nobody else since this is a confirmation page. Would that be an issue?

With security things like this, it's best to have simple reasoning -- e.g. "All content served to users is escaped".  And the "old email" could theoretically have been bad from a previous bug.

Reply:
Can we just HTML-escape the emails before passing them into `SafeString`?

Reply:
Done